torch/_inductor/utils.py

```diff
@@ -977,7 +977,17 @@ def use_triton_template(layout, *, enable_int32=False):
     )
 
 
-def use_cutlass_template(layout):
+def use_cutlass_template(layout, m, n, k):
+    from .virtualized import V
+
+    gemm_size = -1
+    try:
+        # this fails if it's dynamic
+        gemm_size = V.graph.sizevars.size_hint(m * n * k)
+    except Exception:
```

Comment:
Let's make this exception more narrow (if this is still needed).  This could hide bugs.

Reply:
makes sense, I eliminated the try catch entirely..